drivers/soundwire/intel.c

```diff
@@ -1427,6 +1428,16 @@ static int intel_master_startup(struct sdw_master_device *md)
 		pm_runtime_enable(&md->dev);
 	}
 
+	clock_stop_quirks = sdw->link_res->clock_stop_quirks;
+	if (clock_stop_quirks & SDW_INTEL_CLK_STOP_NOT_ALLOWED) {
+		/*
+		 * To keep the clock running we need to prevent
+		 * pm_runtime suspend from happening by increasing the
+		 * reference count
+		 */
+		pm_runtime_get_noresume(&md->dev);
+	}
```

Comment:
@plbossart why not club this with the checks in line 1422 and not even enable runtime  PM?

Reply:
added more explanations.

This is different, the quirk is specified by the parent while disabling pm_runtime is specified by the user for testing only. We want to keep the two separate.